### DISCUSSION TITLE
Development installtions

### DISCUSSION THREAD

**Discussion by Gamer-X** [duckdb/duckdb#12450] [General]
Hi i am trying to setup the development environment for duckdb on windows, https://github.com/duckdb/duckdb/tree/v0.10.2 following this readme. but i thinks its not clear for me how to use it with cmake, i am trying it with mysys64, cloned the repo in mysys, when i try to the command make it works , but when i try the command make debug , it give me error,

"$ make debug
mkdir -p ./build/debug && \
cd build/debug && \
echo  && \
cmake         -DENABLE_EXTENSION_AUTOLOADING= -DENABLE_EXTENSION_AUTOINSTALL= -DLOCAL_EXTENSION_REPO=""  -DOVERRIDE_GIT_DESCRIBE=""  -DDEBUG_MOVE=1 -DCMAKE_BUILD_TYPE=Debug ../.. && \
cmake --build . --config Debug

-- git hash 03bbea5504, version v1.0.1-dev103, extension folder 03bbea5504
-- Extensions will be deployed to: C:/msys64/home/tauru/duckdb/duckdb/build/debug/repository
-- Load extension 'parquet' from 'C:/msys64/home/tauru/duckdb/duckdb/extensions' @ 03bbea5504
-- Extensions linked into DuckDB: [parquet]
-- Configuring done (3.2s)
-- Generating done (0.5s)
-- Build files have been written to: C:/msys64/home/tauru/duckdb/duckdb/build/debug
[1/307] Linking CXX executable duckdb_platform_binary.exe
FAILED: duckdb_platform_binary.exe
C:\Windows\system32\cmd.exe /C "cd . && C:\msys64\mingw64\bin\c++.exe -DDUCKDB_DEBUG_MOVE -g -g -O0 -DDEBUG -Wall    -fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all -Wunused -Werror=vla -Wnarrowing -pedantic  CMakeFiles/duckdb_platform_binary.dir/tools/utils/test_platform.cpp.obj -o duckdb_platform_binary.exe -Wl,--out-implib,libduckdb_platform_binary.dll.a -Wl,--major-image-version,0,--minor-image-version,0  -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cd ."
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lasan: No such file or directory
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lubsan: No such file or directory
collect2.exe: error: ld returned 1 exit status
[18/307] Building CXX object src/planner/binder/...d_statement.dir/ub_duckdb_bind_statement.cpp.obj
ninja: build stopped: subcommand failed.
make: *** [Makefile:274: debug] Error 1
"
anyone can help or give me the link for clear documentation? or should i try on linux

**Comment by Mause** [comment]
```
cannot find -lasan: No such file or directory
cannot find -lubsan: No such file or directory
```
Have you tried having these sanitizers disabled? Or installing them?
